jwt/api_jwt.py

```diff
@@ -126,14 +136,9 @@ def _validate_claims(self, payload, options, audience=None, issuer=None,
             self._validate_aud(payload, audience)
 
     def _validate_required_claims(self, payload, options):
-        if options.get('require_exp') and payload.get('exp') is None:
-            raise MissingRequiredClaimError('exp')
-
-        if options.get('require_iat') and payload.get('iat') is None:
-            raise MissingRequiredClaimError('iat')
-
-        if options.get('require_nbf') and payload.get('nbf') is None:
-            raise MissingRequiredClaimError('nbf')
+        for claim in options.setdefault('require', []):
+            if payload.get(claim) is None:
```

Comment:
I think `options.get('require', [])` is a little more idiomatic than `options.setdefault('require', [])`.